src/runtime-rs/crates/runtimes/virt_container/src/lib.rs

```diff
@@ -104,7 +99,17 @@ impl RuntimeHandler for VirtContainer {
     }
 }
 
-async fn new_hypervisor(_toml_config: &TomlConfig) -> Result<Arc<dyn Hypervisor>> {
-    // TODO: implement ready hypervisor
-    todo!()
+async fn new_hypervisor(toml_config: &TomlConfig) -> Result<Arc<dyn Hypervisor>> {
+    let hypervisor_name = &toml_config.runtime.hypervisor_name;
+    let hypervisor_config = toml_config
+        .hypervisor
+        .get(hypervisor_name)
+        .ok_or_else(|| anyhow!("failed to get hypervisor for {}", &hypervisor_name))
+        .context("get hypervisor")?;
+
+    let mut hypervisor = Dragonball::new();
```

Comment:
@quanweiZhou - How would `runtime-rs` spawn a non-dragonball VMM? This looks like the function that might be able to do that but it doesn't have code to spawn QEMU/CH/ACRN/FC...?

Reply:
@jodh-intel Thanks for your review, try to support other vmm specified by `hypervisor_name`, and already update the code.

```
-    let mut hypervisor = Dragonball::new();
-    hypervisor
-        .set_hypervisor_config(hypervisor_config.clone())
-        .await;
-    Ok(Arc::new(hypervisor))
+    // TODO: support other hypervisor
+    match hypervisor_name.as_str() {
+        HYPERVISOR_DRAGONBALL => {
+            let mut hypervisor = Dragonball::new();
+            hypervisor
+                .set_hypervisor_config(hypervisor_config.clone())
+                .await;
+            Ok(Arc::new(hypervisor))
+        }
+        _ => Err(anyhow!("Unsupported hypervisor {}", &hypervisor_name)),
+    }
```